### PR TITLE
[DAY 2] Day 2 Challenge

### PR DESCRIPTION
**Pull request by Pxyzi3c**

### CODE REVIEW

File: backend/day-02/day-2-input.py [hunk 1]
```diff
@@ -1,2 +1,13 @@
-age = input("Enter your age: ")
-print(f"You're {age} years old.")
\ No newline at end of file
+first_number, second_number, third_number, sum = 0, 0, 0, 0;
```
Comment (theadg) [inline backend/day-02/day-2-input.py]: Do you think writing each initializing each variable on one line each be more readable?

File: backend/day-02/day-2-input.py
Comment (theadg) [inline backend/day-02/day-2-input.py:1]: This is unnecessary but you can "functionalize" your code. E.g., you can put your lines 9-11 even up to line 13 into a function

**Review by theadg — CHANGES_REQUESTED** [review]

File: backend/day-02/day-2-input.py (hunk 1 above)
Comment (Pxyzi3c) [reply backend/day-02/day-2-input.py]: @theadg yezzir

File: backend/day-02/day-2-input.py
Comment (Pxyzi3c) [reply backend/day-02/day-2-input.py:1]: @theadg Thanks for the suggestion but day 1 PR is already merged.

**Review by theadg — APPROVED** [review]

**Comment by theadg** [comment]
Great work! Please Merge 👍

**Pull request merged**